### ISSUE TITLE
Flying mode is not working as expected post QT5.5

### ISSUE THREAD

**Issue by carmenfan** [3drepo/3drepogui#82]
It's no longer repainting when you're flying

**Comment by carmenfan** [comment]
2 things:

- fly mod is working as "intended" when there are more than 1 widgets on the screen and they are linked. Doesn't work as "intended" when they are not linked, or there is 1 widget
- this is returning false @ fly mode, but not for turnable and the other mode: 
````c++
	glcMoverController.hasActiveMover() &&
		glcMoverController.move(GLC_UserInput(x, y));
````
   This determines whether a update() and emit cameraSignal happens, So it is not going into the if condition and yet camera views are changed. mysterious!

**Comment by carmenfan** [comment]
@jozefdobos  i think you fixed it in #126?

**Comment by jozefdobos** [comment]
Yep
